lib/std/os.zig

```diff
@@ -3121,13 +3121,21 @@ pub fn getsockoptError(sockfd: fd_t) ConnectError!void {
     }
 }
 
-pub fn waitpid(pid: i32, flags: u32) u32 {
+
+pub const WaitpidRet = struct {
+   pid: i32,
+   status: u32
+};
+
+pub fn waitpid(pid: i32, flags: u32) waitpid_ret {
```

Comment:
Might as well fix this to use `pid_t` while you're at it
```suggestion
pub fn waitpid(pid: pid_t, flags: u32) WaitpidRet {
```